inc/admin.php

```diff
@@ -22,6 +26,32 @@
 function bootstrap() : void {
 	// Actions.
 	add_action( 'admin_init', __NAMESPACE__ . '\\init_admin_cols', 99 );
+	add_action( 'user_profile_update_errors', __NAMESPACE__ . '\\remove_required_fields_errors', 10, 3 );
+}
+
+/**
+ * Removes required fields errors from the user profile update.
+ *
+ * @param WP_Error $errors WP_Error object (passed by reference).
+ * @param bool     $update Whether this is a user update.
+ * @param stdClass $user   User object (passed by reference).
+ * @return void
+ */
+function remove_required_fields_errors( $errors, $update, $user ) : void {
```

Comment:
Any reason not to add typehints for function params?

```suggestion
function remove_required_fields_errors( WP_Error $errors, bool $update, WP_User $user ) : void {
```

Note need to import WP_User at top of the file.

Reply:
I personally avoid typing filter variables as WordPress isn't the kingdom of reliability, I occasionally had bad surprises doing this.